plugins/contrib/contrib.go

```diff
@@ -0,0 +1,5 @@
+//go:build goprobe_contrib
```

Comment:
In place so that a global `go generate ./...` for the `goProbe` package doesn't involuntary trigger the import of the external repo (`go generate` also supports `-tags` :heart_eyes:).